moc_sprint_tools/sprintman.py

```diff
@@ -62,3 +71,12 @@ def backlog(self):
             raise BoardNotFoundError(self._backlog_name)
 
         return board
+
+    def create_sprint(self, name):
```

Comment:
We should probably log something here when creating a new board. Maybe just move the log message from `create_sprint_boards.py` to here?